src/html_support_files/odoc.css

```diff
@@ -253,17 +253,36 @@ body {
 }
 
 body.odoc {
-  max-width: 100ex;
+  max-width: 122ex;
+  display: grid;
+  grid-template-columns: 28ex 90ex;
```

Comment:
What is the goal of this? Shouldn't the other property already control the width ?

This cause overflow when the window is half the width of my screen.

Reply:
This is how the grid layout usually work, but there this the `fr` unit I should use here for small width cases

Reply:
The grid elements definitely have a size already. Removing this property removes the overflow in my testing.

Reply:
Yes you are in fact right, the content does not have a width, but the sidebar does, and the body has a max witdh.